README.md

```diff
@@ -62,15 +62,99 @@ tags:
   kubernetes_cluster: <string>
   kubernetes_namespace: <string>
   kubernetes_label_ServiceCode: <string>
+
+subscriptionId: subscription-x
+asoAnnotations:
+  serviceoperator.azure.com/reconcile-policy: detach-on-delete
 ```
 
-## Update below resource documentation later 
+### Environment specific Default values 
 
-### NameSpaceQueue
-    In Progress
+Below values are set in flux repositories and all ASO resources will use these values internally. 
 
-### PostgresDatabase
-    In Progress
+for e.g. NameSpace Queues will get created inside `serviceBusNamespaceName` namaspace and postgres database will get created inside `postgresServerName` server.
+
+```
+subscriptionId: <string>                  --subscription Id
+serviceBusResourceGroupName: <string>     --Name of the service bus resource group
+serviceBusNamespaceName: <string>         --Name of the service bus 
+postgresResourceGroupName: <string>       --Name of the Postgres server resource group
+postgresServerName: <string>              --Name of the postgres server
+```
+
+### NameSpace Queue
+
+* Template file: `_namespace-queue.yaml`
```

Comment:
I think you have rebase changes from my feature branch. This changes are already in my PR. Probably you can wait for my PR to merge in main and rebase again to avoid any conflict

Reply:
done